cmd/agent-js-worker/lite/go.mod

```diff
@@ -0,0 +1,117 @@
+// Copyright SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
agent-js-worker is not just wallet, it is an JavaScript agent which has wallet features too (it can be an issuer or verifier too).

So 'lite' is technically an wasm agent which only has wallet features.
It should be called 'vcwallet' instead.